src/components/table/_macro.njk

```diff
@@ -43,7 +43,11 @@
                                             "classes": td.form.button.classes if td.form.button.classes,
```

Comment:
Hi, as we are adding most of the available options to the table button configuration it may be worth here just doing something like 

```
{{
  onsButton(td.form.button)
}}
```

You would then be able to pass in all available parameters to your `form.button {}`

Reply:
I'll try that. Would be good to have an example of that in the examples

Reply:
Sorry, got confused do you mean like that?
```
{{
  onsButton(td.form.button)
}}

```
instead of

```
onsButton({
      "text": td.form.button.text,
      "id": td.form.button.id if td.form.button.id,
      "classes": td.form.button.classes if td.form.button.classes,
      "url": td.form.button.url if td.form.button.url,
      "value": td.form.button.value | safe if td.form.button.value,
      "name": td.form.button.name if td.form.button.name,
      "variants": td.form.button.variants if td.form.button.variants,
      "iconType": td.form.button.iconType if td.form.button.iconType,
      "iconPosition": td.form.button.iconPosition if td.form.button.iconPosition,
      "buttonStyle": td.form.button.buttonStyle if td.form.button.buttonStyle,
  })
```

Reply:
Sounds good to me